packages/rum-core/src/domain/parentContexts.ts

```diff
@@ -97,6 +97,7 @@ export function startParentContexts(lifeCycle: LifeCycle, session: RumSession):
       },
       view: {
         id: currentView!.id,
+        name: currentView!.name,
```

Comment:
That seems a pretty key behavior, could it be interesting to have some tests to avoid regression for it?

Reply:
Done